### PR TITLE
update readme concerning virtual environments

### PR DESCRIPTION
**Pull request by paulstapor**
Just minor add-ons to the readme to make it more foolsafe.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,11 +4,33 @@ This repository contains all python files and (supplementary) figures for the ma
...
+    conda activate solverstudy
+
 All required dependencies can be installed via
 
     pip install -r ../Software/requirements.txt
 
+In the case of an virtual environment, the requirements must be 
+installed to the specific conda environment, i.e.,
+
+    ~/anaconda/envs/solverstudy/bin/pip install -r ../Software/requirements.txt
```
Comment (yannikschaelte) [inline README.md:21]: ususally, `which pip` should already point to the correct one once `conda activate solverstudy` was successful. but full path does not hurt.

File: README.md [hunk 2]
```diff
@@ -4,11 +4,33 @@ This repository contains all python files and (supplementary) figures for the ma
...
+
+or, if miniconda is used instead of anaconda:
+
+    ~/miniconda3/envs/solverstudy/bin/pip install -r ../Software/requirements.txt
+
+Furthermore, since AMICI requires BLAS support and swig, those need to be 
+installed if a new virtual environment was created. This done for anaconda
```
Comment (yannikschaelte) [inline README.md]:
```suggestion
installed if a new virtual environment was created. This is done for anaconda 
```

**Review by yannikschaelte — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (paulstapor) [reply README.md:21]: Tried it, didn't work for me... :/

Comment (yannikschaelte) [reply README.md:21]: hm ... then maybe your conda installation was not properly set up. did you add sth like

```python
# >>> conda initialize >>>
# !! Contents within this block are managed by 'conda init' !!
__conda_setup="$('/home/yannik/anaconda3/bin/conda' 'shell.bash' 'hook' 2> /dev/null)"
if [ $? -eq 0 ]; then
    eval "$__conda_setup"
else
    if [ -f "/home/yannik/anaconda3/etc/profile.d/conda.sh" ]; then
        . "/home/yannik/anaconda3/etc/profile.d/conda.sh"
    else
        export PATH="/home/yannik/anaconda3/bin:$PATH"
    fi
fi
unset __conda_setup
# <<< conda initialize <<<
```
to your .bashrc?

Comment (paulstapor) [reply README.md:21]: Nope, indeed, I didn't...
Good point, thanks!
However: I'd prefer that the correctness of the study relies on things like this to be done manually by the user...

Comment (yannikschaelte) [reply README.md:21]: This is part of what should be done automatically during the anaconda installation, i.e. that it overrides your default python location. If one does not do that, the python env needs to be set manually every time (also make sure that calls to "python" refer to the correct one throughout then!)

**Pull request merged**